pkg/component/shoot/system/system.go

```diff
@@ -444,23 +449,39 @@ func (s *shootSystem) readOnlyRBACResources() []client.Object {
 		})
 	}
 
-	clusterRoleBinding := &rbacv1.ClusterRoleBinding{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:        "gardener.cloud:system:read-only",
-			Annotations: map[string]string{resourcesv1alpha1.DeleteOnInvalidUpdate: "true"},
+	return []client.Object{
+		clusterRole,
+		&rbacv1.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        v1beta1constants.GardenerSystemViewersGroupName,
+				Annotations: map[string]string{resourcesv1alpha1.DeleteOnInvalidUpdate: "true"},
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: rbacv1.GroupName,
+				Kind:     "ClusterRole",
+				Name:     v1beta1constants.GardenerReadOnlyClusterRoleName,
+			},
+			Subjects: []rbacv1.Subject{{
+				Kind: rbacv1.GroupKind,
+				Name: v1beta1constants.GardenerSystemViewersGroupName,
+			}},
 		},
-		RoleRef: rbacv1.RoleRef{
-			APIGroup: rbacv1.GroupName,
-			Kind:     "ClusterRole",
-			Name:     clusterRole.Name,
+		&rbacv1.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        v1beta1constants.GardenerProjectViewersGroupName,
+				Annotations: map[string]string{resourcesv1alpha1.DeleteOnInvalidUpdate: "true"},
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: rbacv1.GroupName,
+				Kind:     "ClusterRole",
+				Name:     v1beta1constants.GardenerReadOnlyClusterRoleName,
+			},
+			Subjects: []rbacv1.Subject{{
+				Kind: rbacv1.GroupKind,
+				Name: v1beta1constants.GardenerProjectViewersGroupName,
+			}},
 		},
-		Subjects: []rbacv1.Subject{{
-			Kind: rbacv1.GroupKind,
-			Name: v1beta1constants.ShootGroupViewers,
```

Comment:
I think it should be safe to remove the `ShootGroupViewers` constant already, or do you have any concerns?

Reply:
Done with 9bdd8fbe96b9f6cb4c8e6941ed4e29907b554152